R/client.R

```diff
@@ -5,7 +5,7 @@ clientForAccount <- function(account) {
 
   if (isShinyappsServer(account$server)) {
     shinyAppsClient(serverUrl, account)
-  } else if (account$server == cloudServerInfo(account$server)$name) {
+  } else if (isCloudServer(account$server)) {
```

Comment:
Note to future us: The `shinyapps.io` server record returns `TRUE` for `isCloudServer`; you may also need `isShinyappsServer` to distinguish from other cloud server records.

Reply:
Annoyances:

1. `account$server` is the name of the server, not a server object. 
1. The `server` variable name can be either a name or an object, depending on context.